.spelling

```diff
@@ -0,0 +1,58 @@
+# markdown-spellcheck spelling configuration file
+# Format - lines beginning # are comments
+# global dictionary is at the start, file overrides afterwards
+# one word per line, to define a file override use ' - filename'
+# where filename is relative to this configuration file
+DSNPArchives
+DSNPMessages
+DSNP
+URIs
+TBD
+blockchain
+Ethereum
+secp256k1
+keccak-256
+DSNPBatch
+dsnpHash
+dsnpUri
+DSNPAnnounce
+ERC165
+testnet
+mainnet
+RLP
+eth2
+CBOR
+GraphChange
+KeyList
+keyList
+PrivateGraphKeylist
+EncryptionKeyList
+messageID
+dsnpData
+inReplyTo
+PrivateGraph
+PrivateGraphChange
+EncryptedInbox
+PrivateBroadcast
+spec
+Dokz
+MDX
+semver
+specs
+enum
+deserialzed
+ArchiveEntries
+bytes32
+archiveEntryID
+dsnpType
+fromAddress
+bytes20
+uri
+deadDropID
+DeadDrops
+actionType
+PrivateGraphKeyList
+toAddress
+utf-8
```

Comment:
UTF-8?

Reply:
I believe this one is case insensitive. I think it treats lowercase as case-insensitive. I should have noted that.